src/schemas/Arranger/initArrangerMetadata/index.ts

```diff
@@ -0,0 +1,82 @@
+import { Client } from '@elastic/elasticsearch';
+import isEqual from 'lodash/isEqual';
+import retry from 'async-retry';
+import { ARRANGER_PROJECT_ID, ARRANGER_FILE_CENTRIC_INDEX } from 'config';
+import logger from 'utils/logger';
+import metadata from 'resources/arranger_es_metadata.json';
+
+export const ARRANGER_PROJECT_METADATA_INDEX = `arranger-projects-${ARRANGER_PROJECT_ID}`;
+export const ARRANGER_PROJECTS_INDEX = `arranger-projects`;
+
+export const FILE_CENTRIC_INDEX = 'file_centric';
```

Comment:
Is this the same as `ARRANGER_FILE_CENTRIC_INDEX` ?

Reply:
👍 good catch! it was, but now it's different, but then it's also not needed anymore :P